docs/docs/events/index.md

```diff
@@ -0,0 +1,71 @@
+# Events
+
+Events refer to the activity in a system that tracee monitors. There are two main types of events, built-in events that are part of Tracee and custom events, which references the way users define other events that Tracee should monitor. 
```

Comment:
activity in a system -> system activity
two main types -> two types
references the way users define other events that Tracee should monitor -> is user defined events